api4/team.go

```diff
@@ -385,7 +387,13 @@ func getTeamMembers(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	members, err := c.App.GetTeamMembers(c.Params.TeamId, c.Params.Page*c.Params.PerPage, c.Params.PerPage, restrictions)
+	excludeDeletedUsersBool, _ := strconv.ParseBool(excludeDeletedUsers)
+	teamMembersGetOptions := &model.TeamMembersGetOptions{
```

Comment:
Since we're touching the API, should we embed `restrictions` into the `TeamMembersGetOptions`? Note that, informally, we leave "required" parameters as first-class, and bury optional parameters into options. Tracing it down, `applyTeamMemberViewRestrictionsFilter` happily accepts an optional restrictions, so I think it could safely be added to this options struct.

Reply:
Done! since `applyTeamMemberViewRestrictionsFilter` accepts a nil restrictions, I think this is a great idea. We can add new options in the future without requiring tons of code change 😄